views/calendar/event.js

```diff
@@ -99,10 +93,3 @@ export default function EventView(props: {eventTitle: string, location: string,
     </View>
   )
 }
-EventView.propTypes = {
```

Comment:
I'm slowly starting to remove runtime PropTypes in favor of Flow. There are babel plugins we can use to automatically generate PropTypes during dev, if anyone's interested.